dnf5/context.cpp

```diff
@@ -667,9 +661,139 @@ std::chrono::time_point<std::chrono::steady_clock> RpmTransCB::prev_print_time =
 
 }  // namespace
 
+
+bool user_confirm_key(libdnf::ConfigMain & config, const libdnf::rpm::KeyInfo & key_info) {
```

Comment:
The function is not listed in the header file. Is it only used in this cpp file? So make it part of anonymous namespace (ended just before this function ) or static.

Reply:
Done.